packages/components/src/public/QueryModel/SaveViewModal.tsx

```diff
@@ -161,31 +162,52 @@ export const SaveViewModal: FC<Props> = memo(props => {
                 <form onSubmit={saveView}>
                     <div className="form-group">
                         <div className="bottom-spacing">
-                            Sort order and filters will be saved as part of custom grid views. Once saved, this view
-                            will be available for all {gridLabel} grids throughout the application. Learn more about{' '}
-                            <HelpLink topic={CUSTOM_VIEW}>custom grid views</HelpLink> in LabKey.
-                        </div>
-                        <div className="bottom-spacing">
-                            <ViewNameInput
-                                onChange={onViewNameChange}
-                                onBlur={onViewNameChange}
-                                view={currentView}
-                                isDefaultView={isDefaultView}
-                            />
+                            Columns, sort order, and filters will be saved. Once saved, this view will be available for
+                            all {gridLabel} grids throughout the application.
                         </div>
+
                         <RequiresPermission perms={PermissionTypes.Admin}>
                             {/* Only allow admins to create custom default views in app. Note this is different from LKS*/}
-                            <div className="form-check bottom-spacing">
-                                <input
-                                    className="form-check-input"
-                                    type="checkbox"
-                                    name="setDefaultView"
-                                    onChange={toggleDefaultView}
-                                    checked={isDefaultView}
-                                />
-                                <span className="margin-left">Make default view for all users</span>
+                            <div className="content-form">
+                                <label className="clickable">
+                                    <input
+                                        className="form-check-input"
+                                        type="radio"
+                                        name="setSaveType"
+                                        id="defaultView"
+                                        onChange={toggleDefaultView}
+                                        checked={isDefaultView}
+                                    />
+                                    <span className="margin-left">Save as default view for all users</span>
+                                </label>
+                            </div>
+                            <div className="content-form">
```

Comment:
I thought we allow non-admin to save custom views?

Reply:
non-admins can save custom views but they can't save their own default views.